dev/halyard_install_component.sh

```diff
@@ -72,11 +88,15 @@ function process_args() {
             shift
             ;;
 
-        --halyard_repository)
-            HALYARD_INSTALL_PARAMS="$HALYARD_INSTALL_PARAMS --halyard-repository $1"
+        --spinnaker_registry)
```

Comment:
Only two of the new flags added to the `--help` seem to be implemented.

Reply:
Or do these get delegated to the python script?

Reply:
They are passed to the install script at line 119

Reply:
Got it.